Havit.Blazor.Components.Web.Bootstrap/Forms/SearchBox/HxSearchBox.razor.cs

```diff
@@ -318,6 +346,62 @@ protected async Task HandleTextQueryValueChanged(string newTextQuery)
 		await InvokeTextQueryChangedAsync(newTextQuery);
 	}
 
+	private async Task UpdateFocusedItem(KeyboardEventArgs keyboardEventArgs)
+	{
+		// Confirm selection on the focused item if an item is focused and the enter key is pressed.
+		TItem focusedItem = GetItemByIndex(focusedItemIndex);
```

Comment:
Do not resolve the `focusedItem` when the key is not the one we are interested in.
In other worlds, resolve it only if it will be used.